client/my-sites/plugins-wpcom/business-plugins-panel.jsx

```diff
@@ -9,7 +9,8 @@ import BusinessPlugin from './plugin-types/business-plugin';
 const defaultPlugins = [
 	{
 		name: 'Google Analytics',
-		supportLink: 'https://en.support.wordpress.com/google-analytics/',
+		supportLink: '/plans/features/google-analytics',
+		SupportLinkTarget: '_self',
```

Comment:
Why is `SupportLinkTarget` capitalized? Also, should we change the name of these attributes, since they are no longer always a `support` link? Maybe `featureDescriptionLink` would be more appropriate for covering both the feature page and the support pages?